packages/react-core/src/demos/ComposableMenu/examples/ActionsMenuDemo.tsx

```diff
@@ -48,7 +30,7 @@ export const ComposableActionsMenu: React.FunctionComponent = () => {
   };
 
   const onToggleClick = (ev: React.MouseEvent) => {
-    ev.stopPropagation(); // Stop handleClickOutside from handling
+    ev.stopPropagation();
     setTimeout(() => {
```

Comment:
We can remove the stopPropagation and the setTimeout since whichever component (Dropdown, Select) should handle focus management already